lib/videoroom/application.ex

```diff
@@ -7,9 +7,14 @@ defmodule Videoroom.Application do
 
   @impl true
   def start(_type, _args) do
+    :ets.new(:room_table, [:named_table, :set, :public])
```

Comment:
Funny enough, it has been in a separate module but I moved it since each function (lookup, instert_new, delete) was called exactly once. I took inspiration from membrane_live btw. where the tables are not wrapped.

Reply:
I don't think that we need concurrency optimizations for the ETS table - imo the reads and writes to the table will be rare - we only read (and create if not present) an entry in the table when creating a new meeting. So unless we are DDoSed by someone repeatedly creating new room, or the backend is connected to many Jellyfishes it shouldn't be a problem.